### PR TITLE
[DESK-1135] send feedback

### PR DESCRIPTION
**Pull request by mauricio0000**
### Description

send feedback UI

### Test plan

1. Sign in
2. Go to lateral menu
3. Click on `Send Feedback`
4. You should see a modal for feedback

### Ticket
https://remoteit.atlassian.net/browse/DESK-1135

### CODE REVIEW

**Comment by mauricio0000** [comment]
Screen Shot 2021-07-13 at 13 34 21

File: frontend/src/models/feedback.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  effects: dispatch => ({
+    async sendFeedback() {
+      const { set } = dispatch.feedback
+
+      set({ sending: true })
+      try {
+        // TO DO
+        // const response = await graphQLSendFeedback({ })
+        // const errors = graphQLGetErrors(response)
+        // if (!errors) {
```
Comment (JamieRuderman) [inline frontend/src/models/feedback.ts:29]: Form should submit to ZenDesk API. Please see ticket description.
https://developer.zendesk.com/api-reference/
Thank you!

**Review by JamieRuderman — CHANGES_REQUESTED** [review]

**Review by JamieRuderman — CHANGES_REQUESTED** [review]
Looks good! It just needs a system message saying "Thank you. Feedback sent!" after sending...
https://remoteit.atlassian.net/browse/DESK-1135

**Review by JamieRuderman — APPROVED** [review]
Thanks!

**Pull request merged**